src/tests/test_experiments.py

```diff
@@ -196,6 +197,17 @@ def test_simple_experiment(experiment_class, matrix_storage_class):
             assert i > 0
         assert len(matrices) == 4
 
+        # 10. Checking that all matrices are associated with the experiment
+        linked_matrices = [
+            row
+            for row in db_engine.execute(
+                    """select * from model_metadata.matrices
+                    join model_metadata.experiment_matrices using (matrix_uuid)
+                    join model_metadata.experiments using (experiment_hash)"""
+            )
+        ]
```

Comment:
this comprehension doesn't do much -- if you like, it can just be:

    list(db_engine.execute(...))